python/lsst/ap/verify/measurements/compute_metrics.py

```diff
@@ -47,18 +47,19 @@ def measure_from_metadata(metadata):
 
     Returns
     -------
-    a list of `lsst.verify.Measurement` derived from `metadata`. May be empty.
+    a list of `lsst.verify.Measurement` derived from ``metadata``. May be empty.
 
     Raises
     ------
-    `RuntimeError`:
-        the config file exists, but does not contain the expected data
+    RuntimeError:
```

Comment:
No colon after the exception in the Raises section

```rst
Raises
------
RuntimeError
    The ``ap_verify`` configuration file exists, but does not contain the
    expected data under ``measurements``.
```

(check the Raises sections of other docstrings)